### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
[DEMO LINK](https://Anton-Zhytonbaiev.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/components/TodoList/TodoList.tsx [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (<TodoInfo key={todo.id} todo={todo} />))}
```
Comment (NVyrsta) [inline src/components/TodoList/TodoList.tsx:11]: Please🙏 add some space to your code
`    {todos.map((todo) => (
      <TodoInfo key={todo.id} todo={todo} />
    ))}`

**Review by NVyrsta — APPROVED** [review]
Good job ;0)

File: src/components/TodoList/TodoList.tsx (hunk 1 above)
Comment (alexandra-protyanova) [reply src/components/TodoList/TodoList.tsx:11]: agree

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Move styles from `App.scss` to appropriate components

**Pull request closed without merging**